src/cluster_legacy.h

```diff
@@ -254,7 +255,8 @@ static_assert(offsetof(clusterMsg, myslots) == 80, "unexpected field offset");
 static_assert(offsetof(clusterMsg, slaveof) == 2128, "unexpected field offset");
 static_assert(offsetof(clusterMsg, myip) == 2168, "unexpected field offset");
 static_assert(offsetof(clusterMsg, extensions) == 2214, "unexpected field offset");
-static_assert(offsetof(clusterMsg, notused1) == 2216, "unexpected field offset");
+static_assert(offsetof(clusterMsg, mflags2) == 2216, "unexpected field offset");
+static_assert(offsetof(clusterMsg, notused1) == 2217, "unexpected field offset");
```

Comment:
Sorry, I just realized that we have 3 **bytes** of mflags, not 3 bits, so we still have 21 bits left to use. So we don't need to introduce mflags 2 here.